Rules/UseCmdletCorrectly.cs

```diff
@@ -129,7 +129,7 @@ private bool MandatoryParameterExists(CommandAst cmdAst)
                 return true;
             }
 
-            if (mandParams.Count() == 0 || Helper.Instance.PositionalParameterUsed(cmdAst))
+            if (mandParams.Count() == 0 || (Helper.Instance.IsCmdlet(cmdAst) && Helper.Instance.PositionalParameterUsed(cmdAst)))

```

Comment:
`!mandParams.Any()` performs in general better than `mandParams.Count() == 0` for `IEnumerable` types because the former stops on the first element but the latter enumerates over all elements. But in this case it is a `List`, therefore we can even just use the property `mandParams.Count == 0`, which does not need to enumerate at all. I know, this code was already like that before, I am not criticising you but we should take the opportunity and make this faster for free.